nevergrad/functions/pyomo/core.py

```diff
@@ -141,9 +144,32 @@ def __init__(self, model: pyomo.Model) -> None:
         exp_tag = ",".join([n.name for n in self.all_objectives])
         exp_tag += "|" + ",".join([n.name for n in self.all_vars])
         exp_tag += "|" + ",".join([n.name for n in self.all_constraints])
-        self.register_initialization(name=exp_tag, model=self._model_instance)
+        self.register_initialization(model=self._model_instance)
+        #self.register_initialization(name=exp_tag, model=self._model_instance)
         self._descriptors.update(name=exp_tag)
 
+    def translate(self, budget):
```

Comment:
this name is not clear, maybe this should be an initialization parameter.
Also, since a copy is made for the function, using the initialization, this translation will actually be ignored